### ISSUE TITLE
Inconsistent field name casing.

### ISSUE THREAD

**Issue by claymcleod** [ga4gh/task-execution-schemas#210]
Ideally, fields would have a common naming convention. The specification currently uses multiple different casing methods in various parts of the spec. For example, the `documentationUrl` field under service information is camel case, while `creation_time` in the task definition is snake case. These should be unified in future versions.

**Comment by vsmalladi** [comment]
Thanks for point this out. Will bring this up in the global API discussions.

**Comment by uniqueg** [comment]
Agreed! We should use consistent naming in 2.0 (and possibly deprecate old names in 1.2).

**Comment by vsmalladi** [comment]
I agree, lets aim to deprecate in 1.2 and no backwards compatibility starting in 2.0. Thoughts @uniqueg

**Comment by uniqueg** [comment]
Yes, exactly what I thought (@vsmalladi). Roughly aiming for 1.2 by the end of the year and 2.0 in 2025, I would say.
